nodejs/src/apis/SessionService.ts

```diff
@@ -0,0 +1,74 @@
+import { APIService } from "../services/APIService";
+import { QueryResultsIterator } from "../utils/QueryResultsIterator";
+import { HMSSession } from "./interfaces/sessionInterfaces";
+
+export class SessionService {
+  private basePath = "/sessions";
+
+  constructor(private apiService: APIService) {}
+
+  /**
+   *
+   * @returns a `QueryResultsIterator<HMSSession>` object
+   */
+  getAllSessionsIterable(): QueryResultsIterator<HMSSession> {
```

Comment:
getSessionsIterable() would be consistent with getRoomIterable

Reply:
also this function itself can take optional filters instead of separate function for filters

```
getSessionIterable()

getSessionsIterable({roomId: ""})
```